src/crush/CrushWrapper.cc

```diff
@@ -2405,13 +2405,13 @@ int CrushWrapper::get_rule_weight_osd_map(unsigned ruleno,
 
   // build a weight map for each TAKE in the rule, and then merge them
 
-  // FIXME: if there are multiple takes that place a different number of
-  // objects we do not take that into account.  (Also, note that doing this
+  // If there are multiple takes that place a different number of
+  // objects we do take that into account.  (Also, note that doing this
   // right is also a function of the pool, since the crush rule
   // might choose 2 + choose 2 but pool size may only be 3.)
```

Comment:
Do you mind rewording the stuff in (Also, ...) think it's really hard to understand.

Reply:
Sure!